src/backoff.erl

```diff
@@ -52,16 +72,30 @@ fire(#backoff{current=Delay, value=Value, dest=Dest}) ->
 -spec get(backoff()) -> pos_integer().
 get(#backoff{current=Delay}) -> Delay.
 
+%% Toggles jitter feature.
+-spec jitter(backoff()) -> backoff().
+jitter(B=#backoff{jitter=false}) ->
```

Comment:
Can we rename to something more descriptive like `init_jitter` or `maybe_seed_jitter` or something? Alternatively have a function like:

```erlang
maybe_seed() ->
case get(random_seed) of
    undefined -> random:seed(erlang:now());
    {X,X,X} -> random:seed(erlang:now());
    _ -> ok
end.
```
That will be cheap to call every time and will even avoid re-seeding an already seeded process.